css/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+#main {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	background-color: #eeeeee;
+}
+#img-div {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	background-color: white;
+	width: 1400px;
+}
+#image {
+	max-width: 1000px;
+	width: auto;
+	height: auto;
+}
+#tribute-info {
+	width: 800px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
```

Comment:
Great job you have done it. 💯 
I would like to advise you to give one line space after each css selectors.